packages/custom_lint/lib/src/analyzer_plugin/lints.dart

```diff
@@ -161,7 +167,11 @@ final allLintsProvider =
         final previousLints = previous?.asData?.value;
         // We voluntarily treat "loading" as null, to clear lints during
         // plugin restart
-        final lints = next.isLoading ? null : next.asData?.value;
+        final lints = next.isRefreshing ||
```

Comment:
What is this condition change supposed to do?

Reply:
It clears lints when the provider refreshes, so that it re-reports the lints in the stdout. Otherwise on a hot-reload it does not give any feedback that it actually reran your linter.

Reply:
Sure, but I'm confused by how this condition achieves it.

First doing `next.isRefreshing || next.isLoading` is redundant as if `isRefreshing` is true, so is `isLoading`

So the core of this change is `(previous?.isRefreshing ?? false)`. But I'll admit I don't understand how this is related to the problem described. Such a condition would only be reached if we did a refresh while we were fetching lints, right?

Reply:
Again, this was something that I added at one point in time because of stdout behavior. It was also with the dev release of riverpod 2.0. I did feel that isRefreshing and isLoading was redundant. Since it has been awhile and things have changed, I do not know if this change is still required.

Reply:
No problem. Thanks for your support even after all this time